apps/financial-aid/backend/src/app/modules/personalTaxReturn/test/municipalitiesPersonalTaxReturn.spec.ts

```diff
@@ -152,9 +148,11 @@ describe('PersonalTaxReturnController - Municipalities Personal Tax Return', ()
 
     it('should return correct values', () => {
       expect(then.result).toEqual({
-        key: signedUrl.key,
-        name: `Framtal_${user.nationalId}_${lastYear}.pdf`,
-        size: Base64.atob(content).length,
+        personalTaxReturn: {
+          key: signedUrl.key,
+          name: `Framtal_${user.nationalId}_${lastYear}.pdf`,
```

Comment:
maybe we should have a function that return this string and takes in nationalid and year, I have seen this string before 👀 ?

Reply:
It used here: https://github.com/island-is/island.is/blob/2f470358935ee0959e92be761f11875f2a6dc11b/apps/financial-aid/backend/src/app/modules/personalTaxReturn/personalTaxReturn.service.ts and then in the this spec file, we could have it as function. But I would probably not call that function in the spec file. 

Do you still want to change it to a function?